### PR TITLE
CMS app is unregistered by defautlt

### PR DESCRIPTION
**Pull request by Domchap90**
# Changes Summary

# Forecast Ticket Link(s)

# FE Screenshots (If Applicable)

# Author Checklist

- [ ]  Can a reviewer skim this PR / Is this a non code change?
- [ ]  Followed DRY (Don't repeat yourself)
- [ ]  Migrations Required/Applied?
- [ ]  Dependency Updates Required
- [ ]  Tests Added
- [ ]  Documentation Updated (If Applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,13 @@ admin.site.register(Author, AuthorAdmin)
 admin.site.register(Tag, TagAdmin)
 ```
 
+## CMS App
+By default this app is not registered as a CMS App. However if you wish to use the builtin CMS app simply add the following line of code to the settings file.
+
+    REGISTER_NEWS_APP = True
+
+Not forgetting to import NewsApp from giant_news.
```
Comment (Will-Hoey) [inline README.md]: Don't need this line here anymore.

**Review by Will-Hoey — COMMENTED** [review]
Can you add a section in the CHANGELOG please, you'll also need to bump the version number

File: pyproject.toml [hunk 2]
```diff
@@ -27,6 +27,8 @@ python = "^3.9"
 giant-mixins = "*"
 django-filer = "*"
 swapper = "^1.3.0"
+flake8 = "4.0.1"
+isort = "^5.10.1"
```
Comment (Will-Hoey) [inline pyproject.toml]: I would say these shouldn't be in the main deps as they will get installed into main deps in projects when this is installed

Comment (Domchap90) [reply pyproject.toml]: yes ofc sorry did that last time i remember now

**Pull request merged**